core/templates/components/summary-tile/topic-summary-tile.directive.html

```diff
@@ -10,16 +10,18 @@
     </div>
     <div class="topic-details" ng-style="{'background-color': $ctrl.getDarkerThumbnailBgColor()}" ng-class="{'mask-effect': !$ctrl.isPublished()}">
       <span class="topic-name"> <[$ctrl.getTopicSummary().getName()]> </span>
-      <span class="topic-story-count">
+      <span class="topic-story-count" ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() > 0">
         <[$ctrl.getTopicSummary().getCanonicalStoryCount()]>
         <span ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() === 1" translate="I18N_TOPIC_VIEWER_STORY"></span>
-        <span ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() !== 1" translate="I18N_TOPIC_VIEWER_STORIES"></span>
```

Comment:
I think we had decided to show the "chapter" count instead. Will you be doing it as part of a separate PR?

Reply:
Yeah, I will be addressing it [here](https://github.com/oppia/oppia/issues/12015).

Reply:
sg, thanks!